client/src/components/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,124 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import axios from 'axios';
+import './RegisterForm.css';
+
+
+const RegisterForm = () => {
+    const {
+        register,
+        handleSubmit,
+        formState: { errors },
+        watch
+    } = useForm();
+
+    const [success, setSuccess] = useState("");
+    const [error, setError] = useState("");
+
+    const createUser = async (username, password, email) => {
+        const newUser = {
+            author: username,
+            password: password,
+            email: email,
+            date: new Date()
+        }
+        const api = axios.create({
+            baseUrl: 'http://localhost:7000'
+        })
```

Comment:
This is bad, if you deploy it on heroku, it's not gonna work. Because it will always contact localhost. You don't even need this at all, delete it and use axios.post or axios.get, etc...